fms/testing/_internal/hf/model_test_suite.py

```diff
@@ -140,15 +140,17 @@ def test_hf_model_compile_no_graph_breaks(self, fms_hf_model):
         try:
             compiled_model = torch.compile(fms_hf_model, fullgraph=True)
             fms_hf_signature_params = HFModelSignatureParams(
```

Comment:
why is this `HF` model signature params? I think the relevant model we're testing in this PR is an FMS model?

Reply:
This test is testing the hf adapted version. This would be a good test for this as this is how it is being called there.

Reply:
where? the test below that we're trying to fix is in `ModelCompileTestSuite(ModelFixtureMixin)`, no reference to HF, and not in the /hf/ directory... If it was testing compilation of the HF adapted one, wouldn't it have to have been `model.decoder = torch.compile(model.decoder)` ?

Reply:
That is only required when calling generate, If you call the model itself, which TGIS is doing, its just a normal torch compile.

Reply:
This is hf/model_test_suite.py HFModelCompileTestSuite